src/com/ms/silverking/cloud/dht/daemon/MessageModule.java

```diff
@@ -497,7 +514,7 @@ private void handlePutUpdate(MessageGroup message, MessageGroupConnectionProxy c
         version = ProtoPutUpdateMessageGroup.getPutVersion(message);
         storageState = ProtoPutUpdateMessageGroup.getStorageState(message);
         if (debug) {
-            System.out.println("handlePutUpdate storageState: "+ storageState);
+            Log.fineAsyncf("handlePutUpdate storageState: %s ", storageState);
```

Comment:
debug is off by default. Make Log.warningAsyncf so that the debug flag has an effect
Similar cases in the rest of this file